action.yml

```diff
@@ -32,13 +32,24 @@ inputs:
     description: 'Do not verify certificates during download'
     required: false
     default: 'false'
+  base_branch:
+    description: 'The branch into which you want the schema merged. If not set, will use the default branch'
+    required: false
   branch:
     description: 'The name of the branch to use to create the pull request. If not set, will use update-schema-MM-dd-HH:mm'
     required: false
   remote:
     description: 'The remote to push the branch used to create the pull request'
     required: false
     default: 'origin'
+  pr_title:
+    description: 'The title of the pull request that is created'
+    required: false
+    default: 'Update GraphQL Schema'
+  pr_body:
+    description: 'The body of the pull request that is created'
+    required: false
+    default: 'Update GraphQL Schema'
```

Comment:
👍  would be nice to include a summary of the changes (number of additions/deletions, etc...) at some point